File: src/CoMute/ClientApp/src/types/carPool.ts
export interface CarPool {
  id: number;
  departureTime: string;
  arrivalTime: string;
  origin: string;
  daysAvailable?: any; 
  destination: string;
  availableSeats: number;
  ownerId: number;
  created: string;
  notes?: string;
}

File: src/Models/Carpool.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FSWebApi.Models
{
    public class Carpool
    {
        [Key]
        public Guid CarpoolId { get; set; }
        public string Origin { get; set; }
        public string Destination { get; set; }
        public DateTime DayAvailable { get; set; }
        public TimeSpan DepartureTime { get; set; }
        public TimeSpan ArrivalTime { get; set; }
        public string? Notes { get; set; }
        public int Seats { get; set; }    
        public Guid OwnerID { get; set; }
        public DateTime DateCreated { get; set; }
        public virtual ICollection<CarpoolMember>? Members { get; set; }



    }
}
